sources/packages/api/src/route-controllers/education-program-offering/models/education-program-offering.dto.ts

```diff
@@ -124,3 +124,23 @@ export class OfferingAssessmentAPIInDTO {
   @IsNotEmpty()
   assessmentNotes: string;
 }
+
+/**
+ * DTO to display offering which is requested for change.
```

Comment:
I personally prefer to avoid starting the comments stating what is being commented on, for instance, "DTO to", "Method for", "Endpoint to do", "Class to", etc.